resources/data/defaultpresets.xml

```diff
@@ -2844,7 +2844,7 @@
             <reference ref="link_contact_address_payment" />
             <preset_link preset_name="Building" />
         </item> <!-- Fuel -->
-        <item name="Charging Station" icon="presets/vehicle/charging_station.svg" type="node" preset_name_label="true">
+        <item name="Charging Station" icon="presets/vehicle/charging_station.svg" regions="US,CA" type="node" preset_name_label="true">
```

Comment:
This isn't actually limited to the United States and Canada. We'll have to remember to drop this prior to merge.
```suggestion
        <item name="Charging Station" icon="presets/vehicle/charging_station.svg" type="node" preset_name_label="true">
```

Reply:
You could (/should) just change it locally without committing the change to git

Reply:
That seems like the better option.

Sorry, will keep that in mind.

Reply:
The easy solution for this is to use a custom preset for testing / dev purposes and simply disable the default preset.